### DISCUSSION TITLE
`display:none` element and steps

### DISCUSSION THREAD

**Discussion by ShirasuGyoza** [gilbarbara/react-joyride#827] [Q&A] [answered]
Hi, I have a question about step target of `display: none` . I'm creating app and there are some elements that may hidden according to user type so that element's css is `display: none`.  And I want to set steps target to this element.

If steps target is `display:none`, the overlay disappear after the step of hidden element. 
How can I fix it? Is there any option or solution to skip target if the element is hidden?

**Comment by ShirasuGyoza** [comment]
seems the same as: #519
any updates?

**Comment by gilbarbara** [comment, marked as answer]
hey @ShirasuGyoza 

You can filter out the steps for elements that you know aren't visible to avoid the issue you mentioned.
